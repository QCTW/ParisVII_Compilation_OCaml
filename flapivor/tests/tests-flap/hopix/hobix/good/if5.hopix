../../../../tests-m2/hopix/interpreter/good/if5.hopix