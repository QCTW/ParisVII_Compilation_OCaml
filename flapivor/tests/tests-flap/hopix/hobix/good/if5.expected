../../../../tests-m2/hopix/interpreter/good/if5.expected